libraries/stdlib/js-ir/builtins/Arrays.kt

```diff
@@ -53,17 +59,6 @@ public class CharArray(size: Int) {
     public operator fun iterator(): CharIterator
 }
 
-@kotlin.internal.InlineOnly
```

Comment:
@pyos Seems this removal causes next fail: https://teamcity.jetbrains.com/viewLog.html?buildId=2236246&tab=buildResultsDiv&buildTypeId=Kotlin_dev_Compiler
//cc @anton-bannykh

Reply:
@max-kammerer Those tests are built with bootstrap compiler. Builds successfully with bootstrap.local=true. Should be fixed by advancing the version.